### DISCUSSION TITLE
drag-canvas 能够修改成鼠标中键拖动画布吗？

### DISCUSSION THREAD

**Discussion by AndySpider** [antvis/G6#4718] [Q&A]
默认是鼠标左键，想改成中键按住拖动

**Comment by Jalilvand-MJ** [comment]
Any responses?
Changing this line might solve the problem.
https://github.com/antvis/G6/blob/a73b940d87fa0eb04eea4bfad7fb70cb3aaf81e2/packages/pc/src/behavior/drag-canvas.ts#L121
